tests/store/test_find_tag.py

```diff
@@ -0,0 +1,28 @@
+"""Tests for finding tags in store."""
+from housekeeper.store import Store
+from housekeeper.store.models import Tag
+
+
+def test_tag_with_tag_name(populated_store: Store, sample_tag_name: str):
+    """Test fetching a tag from database given a tag name."""
+    # GIVEN a populated store and a tag name
+
+    # WHEN retrieving a tag from Store
+    test_tag = populated_store.tag(tag_name=sample_tag_name)
+
+    # THEN a tag should be returned
+    assert isinstance(test_tag, Tag)
+
+    # THEN the tag should have the tag name gotten as parameter
+    assert test_tag.name == sample_tag_name
+
+
+def test_tag_without_tag_name(populated_store: Store):
```

Comment:
Should a tag really be returned if no tag name is provided? I would assume it should return None but I'm not sure :)

Reply:
to discuss tomorrow in meeting